src/app.js

```diff
@@ -457,9 +457,7 @@ class Messenger extends EventEmitter {
   /**
    * Send a response to the default page
    *
-   * **DEPRECATED** 1.4.0 Use :meth:`Response.reply` instead
-   *
-   * .. depreacted:: 1.4.0
```

Comment:
haha this never worked because I can't spell

Reply:
Depreacted – the new React framework :-)